contracts/.gas-snapshot

```diff
@@ -1,30 +1,30 @@
-OmniPortal_xsubmit_Test:test_xsubmit_duplicateValidator_reverts() (gas: 13331351)
-OmniPortal_xsubmit_Test:test_xsubmit_invalidAttestationRoot_reverts() (gas: 13377290)
-OmniPortal_xsubmit_Test:test_xsubmit_invalidMsgs_reverts() (gas: 13364249)
-OmniPortal_xsubmit_Test:test_xsubmit_noQuorum_reverts() (gas: 13338273)
-OmniPortal_xsubmit_Test:test_xsubmit_wrongChainId_reverts() (gas: 13392048)
-OmniPortal_xsubmit_Test:test_xsubmit_wrongStreamOffset_reverts() (gas: 13390276)
-OmniPortal_xsubmit_Test:test_xsubmit_xblock1_chainB_succeeds() (gas: 13590922)
-OmniPortal_xsubmit_Test:test_xsubmit_xblock1_succeeds() (gas: 13591783)
-OmniPortal_xsubmit_Test:test_xsubmit_xblock2_chainB_succeeds() (gas: 13863704)
-OmniPortal_xsubmit_Test:test_xsubmit_xblock2_succeeds() (gas: 13865465)
\ No newline at end of file
+FeeOracleV1_Test:test_feeFor_succeeds() (gas: 35506)
+FeeOracleV1_Test:test_setFee_nonOwner_reverts() (gas: 18013)
+FeeOracleV1_Test:test_setFee_succeeds() (gas: 28284)
+FeeOracleV1_Test:test_setFee_zero_reverts() (gas: 18054)
+OmniPortal_admin_Test:test_setFeeOracle_nonOwner_reverts() (gas: 18109)
+OmniPortal_admin_Test:test_setFeeOracle_succeeds() (gas: 29286)
+OmniPortal_admin_Test:test_setFeeOracle_zero_reverts() (gas: 18211)
+OmniPortal_exec_Test:test_exec_aheadOffset_reverts() (gas: 23720)
+OmniPortal_exec_Test:test_exec_behindOffset_reverts() (gas: 172646)
+OmniPortal_exec_Test:test_exec_wrongChainId_reverts() (gas: 21069)
+OmniPortal_exec_Test:test_exec_xmsgRevert_succeeds() (gas: 135604)
+OmniPortal_exec_Test:test_exec_xmsg_succeeds() (gas: 184076)
+OmniPortal_feeFor_Test:test_feeFor_defaultGasLimit_succeeds() (gas: 28916)
+OmniPortal_feeFor_Test:test_feeFor_succeeds() (gas: 27888)
+OmniPortal_xcall_Test:test_xcall_defaultGasLimit_succeeds() (gas: 79787)
+OmniPortal_xcall_Test:test_xcall_explicitGasLimit_succeeds() (gas: 80357)
+OmniPortal_xcall_Test:test_xcall_gasLimitTooHigh_reverts() (gas: 46270)
+OmniPortal_xcall_Test:test_xcall_gasLimitTooLow_reverts() (gas: 46282)
+OmniPortal_xcall_Test:test_xcall_insufficientFee_reverts() (gas: 44818)
+OmniPortal_xcall_Test:test_xcall_sameChain_reverts() (gas: 46188)
+OmniPortal_xsubmit_Test:test_xsubmit_duplicateValidator_reverts() (gas: 13337260)
+OmniPortal_xsubmit_Test:test_xsubmit_invalidAttestationRoot_reverts() (gas: 13383151)
+OmniPortal_xsubmit_Test:test_xsubmit_invalidMsgs_reverts() (gas: 13370182)
+OmniPortal_xsubmit_Test:test_xsubmit_noQuorum_reverts() (gas: 13344093)
+OmniPortal_xsubmit_Test:test_xsubmit_wrongChainId_reverts() (gas: 13402409)
+OmniPortal_xsubmit_Test:test_xsubmit_wrongStreamOffset_reverts() (gas: 13396233)
+OmniPortal_xsubmit_Test:test_xsubmit_xblock1_chainB_succeeds() (gas: 13601419)
+OmniPortal_xsubmit_Test:test_xsubmit_xblock1_succeeds() (gas: 13602679)
+OmniPortal_xsubmit_Test:test_xsubmit_xblock2_chainB_succeeds() (gas: 13879461)
+OmniPortal_xsubmit_Test:test_xsubmit_xblock2_succeeds() (gas: 13882068)
```

Comment:
How was this snapshot generated?

Reply:
We use `forge snapshot`. It is run during pre-commit